.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,113 @@
+name: E2E Testing
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: ["ready_for_review", "opened"]
+
+env:
+  AWS_DEFAULT_REGION: us-east-1
+  AWS_DNS_ROLE: "arn:aws:iam::821755571104:role/aptible-cloud-staging-dns-cross-account"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    environment: cloud_staging
+
+    strategy:
+      fail-fast: false
+
+      matrix:
+        include:
+          - test: acm-dns-validated-create
+            env_id: "5bb16655-4102-49f1-88c8-20f198fc139a"
+            aws_account_id: "395100626517"
+
+          - test: acm-non-validated-create
+            env_id: "3991abc1-e98e-4f4f-8de1-66e8d7e0c5dd"
+            aws_account_id: "309087580932"
+
+          - test: ecs_compute_create
+            env_id: "18f97b0c-7054-4041-a064-6cc46c241cb3"
+            aws_account_id: "100952742100"
+
+          - test: ecs_compute_update
+            env_id: "23f58fa1-c190-4f82-a68c-fd1b56b2cbfe"
+            aws_account_id: "759853141917"
+
+          - test: ecs_web_create
+            env_id: "e675979c-cb76-4b78-b6e2-dd4ca7b91342"
+            aws_account_id: "692148248699"
+
+          - test: elasticache_create
+            env_id: "abe891e3-4e08-43af-8525-b290bb175f84"
+            aws_account_id: "917738356426"
+
+          - test: elasticache_update
+            env_id: "03440b0b-9b84-4004-80c9-5cdc6e768f8e"
+            aws_account_id: "327177179188"
+
+          - test: rds
+            env_id: "e9a067d9-8fa6-44c9-9a80-cf76aca8b878"
+            aws_account_id: "740682447574"
+
+          - test: secret_create
+            env_id: "413f7b2f-6d73-4412-8828-ac49f595efd3"
+            aws_account_id: "908740948823"
+
+          - test: secret_update
+            env_id: "f3363014-0b95-47a4-97b4-a5b92fb1c005"
+            aws_account_id: "453862071053"
+
+          - test: vpc_create
+            env_id: "e4eecaee-6ce5-4915-92d0-98a9fe036ab4"
+            aws_account_id: "257348118889"
+
+          - test: vpc_update
+            env_id: "89ca29f6-e245-4ac0-b796-1943d65d5933"
+            aws_account_id: "604856548006"
+
+    # These permissions are needed to interact with GitHub's OIDC Token endpoint.
+    permissions:
+      id-token: write
+      contents: read
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_version: ~1.3.0
```

Comment:
i think we might have a list somewhere where we keep track of it but this is starting to expand and easily missed. there's a few places i noticed we have older versions (only noticed on cloud-dev once) that were seemingly inconsequential (I think i saw a ~1.2.x reference somewhere)

Reply:
Since this is public facing I think we should ultimately turn this into a matrix option for all of our supported versions, but that takes a bit of effort around the account management stuff. Spacelift has added the ability to set ranges so I updated our prod repo to always use 1.3.x for now. Outside of that our containers are all on 1.3.4 and I plan on updating them to 1.3.5 today (it came out yesterday).